src/lsp/cobol_indent/cobol_indent.ml

```diff
@@ -16,10 +16,62 @@ module Type = Indent_type
 (*return the result of indentation. use user-defined indent_config*)
 let indent_range = Indenter.indent_range
 
-(*indent the whole file and print*)
-let indent_file ~dialect ~source_format ~file ~indent_config =
-  let contents = Ez_file.V1.EzFile.read_file file in
-  indent_range
-    ~dialect ~source_format ~range:None ~indent_config
-    ~filename:file ~contents
-  |> Fmt.pr "%s"
\ No newline at end of file
+(** [apply s rdl] applies the changes in [rdl] to the text in [s]. [rdl] is
+    expected to have been computed by calling [indent_range ~contents:s]. *)
+let apply =
```

Comment:
If you don't name the arguments here, could you add the type annotation? Then they'd stand out better than in the comment.

Reply:
Sure!